### DISCUSSION TITLE
AQ20 - Fight with Ayamiss

### DISCUSSION THREAD

**Discussion by Plasmashield** [chromiecraft/chromiecraft#3974] [PTR Feedback]
I suggest reducing the health of the [Hive'Zara Larva](https://wowgaming.altervista.org/aowow/?npc=15555). They currently have 30,000 hit points, which is too much. The fight is designed in such a way that only melee should attack them. Originally, they had about 3k health. Perhaps it should be around 5-10k, so as not to break the fight by making the ranged attack the larvae

**Comment by Annamaria-CC** [comment]
Hey, we had a recent tests with an split amount of ranged/melee and it is totally doable by just using your melee on it. We will keep monitoring to see if any adjustments need to be made.
